### ISSUE TITLE
Help output in repl

### ISSUE THREAD

**Issue by cmaujean** [erikh/box#201]
While exploring in the repl, I'd like to be able to type 'help' and get a list of box commands. i'd like to be able to type `help <command>` and get a description and usage on that command. much like the web docs, but in the repl. This should, I think, respect the --omit commandline arg and not show help for unavailable commands. 

I will be submitting a patch for this, unless the idea gets totally shot down.

**Comment by erikh** [comment]
Please do make this patch!

**Comment by erikh** [comment]
One suggestion: do it in the repl parse phase, not at the ruby call level,
e.g. More like "quit" and "exit" and not like "from". This means it won't
have to conform to string handling rules in ruby, and will be easier to use
in general.

**Comment by cmaujean** [comment]
yeah, I hadn't even considered doing it in the ruby phase. It's just not a thing that could logically go at that low of a level.

**Comment by erikh** [comment]
partially resolved by #235 and #239
